pkg/kapp/resources/mod_waiting_rules.go

```diff
@@ -0,0 +1,30 @@
+package resources
```

Comment:
this file should move into https://github.com/k14s/kapp/tree/master/pkg/kapp/resourcesmisc.

Reply:
Why? Doing that creates an import cycle as this file depends on items from the resources package.

Reply:
Nevermind. I had some old code (your comment above)